src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsNumberAttendeesMappingTest.java

```diff
@@ -232,4 +239,21 @@ void testGetNumberOfOtherPartyAttendees() {
 
         assertThat(result).isEqualTo(2);
     }
+
+    @DisplayName("When otherParties are given and other than face-to-face hearing, getNumberOfAppellantAttendees returns the count of those that want to attend")
```

Comment:
Is there supposed to be a otherParties property? Also not seeing where the hearing type is defined as not-f2f.

Reply:
The description of the test case is wrong, updated with the proper description as per the test case.